Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/CWFS.rst

```diff
@@ -4,11 +4,94 @@
 
 .. _AT-On-sky-CWFS:
 
-#######################
-CWFS
-#######################
+#################################
+Focus, center and absorb offsets
+#################################
+
+.. _Beginning_of_the_night-the-Telescope-Overview:
+
+Overview
+========
+
+This procedure must be followed by the observers before the beginning of dark time, prior to starting observations. 
+It will focus the telescope, center a target and absorb the pointing offsets. 
+
+These steps are also required everytime the ATCS transitions to standby for whatever reason, and is re-enabled to continue observing. 
```

Comment:
I don't think this is actually necessary. The components will keep their previous state when they go to standby and back, so we're probably good doing this only at the start of the night.

Reply:
When ATCS transitions to STANDBY, the dome and mirror cover close, the ATAOS corrections disable, and we need to run prepare_for_sky. Do you confirm that the ATAOS corrections are saved, don't you?